### PR TITLE
remove react-mdl

### PR DESCRIPTION
**Pull request by mikbry**
Remove all dependencies to react-mdl

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -50,7 +49,7 @@
     "style-loader": "^0.19.1",
     "zoapp-common": "0.1.2",
     "zoapp-materialcomponents": "^0.2.1",
-    "zoapp-ui": "0.1.5"
+    "zoapp-ui": "0.2.0"
```
Comment (willdurand) [inline package.json]: This might introduce a regression related to the dialogs/dialog manager

Comment (mikbry) [reply package.json]: how ?

Comment (willdurand) [reply package.json]: I think there are still files where we import `DialogManager` from `zoapp-ui`:

```
import { DialogManager } from "zoapp-ui";
```

and I dont think we have it anymore in `zoapp-ui:0.2.0`

Comment (mikbry) [reply package.json]: We don't have to use DialogManager, but I will export it ok.

Comment (willdurand) [reply package.json]: Yes I know but by keeping `zoapp-ui:0.1.5` we can progressively migrate. Here that breaks everything until we use `Rmdc.showDialog()` everywhere :D

Comment (mikbry) [reply package.json]: I will fully migrate Opla/front to Rmdc.showDialog() tomorrow ;-) And Opla/front rely on zoapp-front 0.1.2. I will update it to 0.2.0 today.

Comment (mikbry) [reply package.json]: I exported DialogManager from zoapp-materialcomponents for emergency cases. Like your displayWebServiceEditor ;-)

**Pull request merged**